recipes/booster/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: booster
+  version: {{ version }}
+
+
+source:
+  url: https://github.com/evolbioinfo/booster/archive/v{{ version }}.tar.gz
+  sha256: eee92b91c98d3fccb2b94201dfdc775d1c381d22c5953240562b8de20e763e18
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - llvmdev  # [osx]
+    - libcxx 4.0.*  # [osx]
```

Comment:
try adding `llvm-openmp` to the `build:` and `run:` sections. You can then remove `openmp` (actually, you should be able to remove the entirety of what's now in your `host:` and `run:` sections).